res/values-es/strings.xml

```diff
@@ -35,39 +35,39 @@
     <string name="LeftSettings">Opciones</string>
     <string name="LeftStories">Historias</string>
     <string name="Loading">Cargando&#8230;</string>
-    <string name="MoreMenuTitle">Opciones de la Página</string>
+    <string name="MoreMenuTitle">Opciones de la página</string>
     <string name="NewSources">¡Hemos agregado nuevas fuentes de historias!</string>
     <string name="NewSourcesNoThx">Luego</string>
     <string name="NewSourcesOk">Muéstrame</string>
     <string name="No">No</string>
-    <string name="NoChange">Ningún cambio</string>
-    <string name="OnlyOnPhone">Solo en Teléfono</string>
-    <string name="OpenInExternalBrowser">Abrir en navegador externo</string>
+    <string name="NoChange">Sin cambios</string>
+    <string name="OnlyOnPhone">Solo en el teléfono</string>
+    <string name="OpenInExternalBrowser">Abrir en el navegador externo</string>
     <string name="OrbotCheckSite">https://check.torproject.org</string>
-    <string name="OrbotStatusOkMessage">!Orbot parece estar corriendo bien!</string>
-    <string name="OrbotStatusOkTitle">Orbot estatus </string>
-    <string name="PageOptionsTitle">Opciones de Pagina</string>
-    <string name="PrefAbout">Acerca</string>
+    <string name="OrbotStatusOkMessage">¡Orbot parece estar corriendo bien!</string>
+    <string name="OrbotStatusOkTitle">Estado de Orbot </string>
+    <string name="PageOptionsTitle">Opciones de la página</string>
+    <string name="PrefAbout">Acerca de</string>
     <string name="PrefAcraDisabled">Reporte automático desactivado</string>
```

Comment:
"Informe" is a much more proper word in Spanish than "Reporte", so: "Informe automático desactivado" would be better.